src/utils/GridConnectHub.cxx

```diff
@@ -404,8 +404,8 @@ struct GcHubPort : public Executable
     {
         LOG(VERBOSE, "gchub port %p", (Executable *)this);
         if (use_select) {
-#ifdef ARDUINO
-            DIE("select is not supported on Arduino");
+#ifdef OPENMRN_FEATURE_EXECUTOR_SELECT
```

Comment:
Shouldn't this be ifndef?

Reply:
Good catch, thanks for looking carefully!